examples/CountLog.hs

```diff
@@ -139,7 +140,7 @@ data CountLogCtx = CountLogCtx
 
 
 -- | Deriving two capabilities from the record fields of @MonadReader@.
-newtype CountLogM m (a :: *) = CountLogM (ReaderT CountLogCtx m a)
+newtype CountLogM m (a :: Type) = CountLogM (ReaderT CountLogCtx m a)
```

Comment:
We should also use toplevel kind signatures. But let's worry about that another day.